full translation of c00 (remaining c00/30/21->32, c00/34, c00/50) (#72)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/50/25/15/translation.txt

//
// Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
//

> 少々罰当たりな気もしますが、そこが興奮に繋がりそうですね。……そう思っているのは私だけでしょうか？=It feels a bit condemnable to do it here, but this is what makes it exciting, right? ...Am I the only one who think so?

Reviewer: ```suggestion
少々罰当たりな気もしますが、そこが興奮に繋がりそうですね。……そう思っているのは私だけでしょうか？=It feels wrong to do it here, but this is what makes it exciting, right? ...Am I the only one who think so?
```

> 結婚式の予行をするのですか？　私との未来を考えてくださり嬉しく思います。=Are we going to do a wedding rehearsal? I'm glad that you're thinking about our future.

Reviewer: ```suggestion
結婚式の予行をするのですか？　私との未来を考えてくださり嬉しく思います。=Are we having a wedding rehearsal? I'm glad that you're thinking about our future.
```

本物ではありませんが、神聖な雰囲気がありますね。そんな場所でセックスをするなんて、私達はなんて罪深いのでしょう。=It may not be real, but it still has a sacred atmosphere. How sinful of us to have sex in such a place.

> 随分と罰当たりなことをしたがるのですね。これ以上罪を重ねてどうするんですか？=You want to do something that's quite punishable. How will you atone for many sins?

Reviewer: Wedding chapel event is all as fem-dom, so I think it's more like this:

```suggestion
随分と罰当たりなことをしたがるのですね。これ以上罪を重ねてどうするんですか？=You're so eager for me to punish you. Are you going just keep sinning?
```

Author: i think she's a bit straight-forward here... disgust intimacy lvl, so she's presuming and condemning the fact that he wants to do it in a "holy" place. (and probably will agree to punish him as atonment) :P 

your suggestion makes sense, but only if it takes place after the guy proposes she dominates him, and going by the other lines, it seems the sex discussion hadn't even taken place yet.  i'll leave it as it is for now, feels like this needs a bit more thought

Reviewer: I think I may have missed the point here, it's not about her punishing him, but the fact that he's already going to hell even without screwing in church
```suggestion
随分と罰当たりなことをしたがるのですね。これ以上罪を重ねてどうするんですか？=You're already damned for all eternity. Are you going to keep piling on more sins?
```

Author: yep, i think this is more inline with what i understood as well
